package.json

```diff
@@ -69,8 +73,9 @@
     "@angular-devkit/core": "^10.0.0-rc.2",
     "@angular-devkit/schematics": "^10.0.0-rc.2",
     "@angular/bazel": "^10.0.0-rc.2",
+    "@angular/benchpress": "^0.2.0",
     "@angular/compiler-cli": "^10.0.0-rc.2",
-    "@angular/dev-infra-private": "https://github.com/angular/dev-infra-private-builds.git#2ac83eb462cb25c46a761d34dec030e360055016",
+    "@angular/dev-infra-private": "https://github.com/angular/dev-infra-private-builds#master",
```

Comment:
I should probably change this to a different commit hash

Reply:
Any reason why this is not done yet? We should use a specific SHA.

Reply:
Looks like it's done! 🎉 (my comment was still old in the pipeline)